management/azure_mgmt_compute/spec/virtual_machines_spec.rb

```diff
@@ -135,7 +135,7 @@
 
     result = @resource_helper.network_client.network_interfaces.list_all
     result.each do |network_interface|
-      expect(network_interface.virtual_machine).to be_an_instance_of(MsRestAzure::SubResource)
+      expect(network_interface.virtual_machine).to be_an_instance_of(VirtualMachine)
```

Comment:
you should have to write `Azure::ARM::Compute::Models::VirtualMachine`

Reply:
Actually, it should be Azure::ARM::Network::Models::SubResource. The model looks like this:

```
class NetworkInterface < Resource

      include MsRestAzure

      # @return [SubResource] The reference of a virtual machine.
      attr_accessor :virtual_machine
```

So, changed it with the latest commit